Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -109,6 +109,12 @@ uplink-storage-implanter-desc = Hide goodies inside of yourself with new bluespa
 uplink-freedom-implanter-name = Freedom Implanter
 uplink-freedom-implanter-desc = Get away from those nasty sec officers with this three use implant!
 
+uplink-dna-scrambler-implanter-name = DNA Scrambler Implanter
+uplink-dna-scrambler-implanter-desc = A single use implant that can be activated to modify your DNA and give you a completely new look, also has the function of undoing the change. Unable to scramble already scrambled DNA.
```

Comment:
Please also add a separate popup text in here saying that they can no longer scramble DNA.

Reply:
I added a popup for when people attempt to use a DNA scrambler implant while already transformed. I also made the DNA scrambler implanter not break after implanting, this allows someone to extract it if they accidentally implant their scrambled form with it.